Tasks/KubernetesManifestV0/src/utils/KubernetesObjectUtility.ts

```diff
@@ -205,7 +205,20 @@ export function getResources(filePaths: string[], filterResourceTypes: string[])
         const fileContents = fs.readFileSync(filePath);
         yaml.safeLoadAll(fileContents, function (inputObject) {
             const inputObjectKind = inputObject ? inputObject.kind : '';
+            let inputObjectStrategyType = '';
+            if (inputObject && inputObject.spec && inputObject.spec.updateStrategy) {
+                inputObjectStrategyType = inputObject.spec.updateStrategy.type;
+            }
```

Comment:
Set else to `RollingUpdate` maybe?

Reply:
Add the check from line # 215 here. And remove it from there. And set the inputObjectStrategyType to `RollingUpdate` only if it's a workload. That should simplify the logic there.

Reply:
Done.